main.cpp

```diff
@@ -0,0 +1,419 @@
+#include <SFML/Graphics.hpp>
+#include <SFML/Network.hpp>
+#include <iostream>
+#include <windows.h>
+#include <map>
+#include "TextureHolder.h"
+#include "Square.h"
+#include "Piece.h"
+
+bool move(Square* slctd, Square* trgt, sf::UdpSocket& sckt, sf::IpAddress& IP, unsigned short& PORT, sf::Packet& pck);
+
+int main()
```

Comment:
## Nie można wszystkiego w do main wrzucić
C++ jest obiektowy -> używamy klas.

Tu powinna być, co najmniej, jakaś klasa typu: "GameController", która robiła by *wszystko* co main.
Jeszcze lepiej, kilka klas, każda robiąca 'swoją' rzecz - inicjalizacja okna nie jest związana z otrzymywaniem ruchów (to jest przykład, nie cały problem)

Reply:
Wtedy znika problem 'kolejności' zmiennych w pliku - były by one memberami jakiejś innej klasy